pom.xml

```diff
@@ -80,6 +80,28 @@ under the License.
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>shade-sources</id>
+            <activation>
+                <property>
+                    <name>shade-sources</name>
+                </property>
```

Comment:
Why do we have to enable the profile via a property? Wouldn't `-Pshade-sources` be enough?

Reply:
An activation property is a lot easier to work with in scripts and stuff, see https://issues.apache.org/jira/browse/FLINK-7850.

Reply:
Alright, thanks for the clarification.